StudentDataAnalysator/ViewModels/ViewExcelViewModel.cs

```diff
@@ -59,20 +64,26 @@ public int TableType
             }
         }
 
-        private void LoadTable(string path)
+        private void GetTableType(string path)
         {
             ExcelFileLoaderService _excelDataReader = new ExcelFileLoaderService(path);
 
             if (_excelDataReader.GetTableType() == (int)TableTypeEnum.StudentsResultTable)
-            {
                 TableType = (int)TableTypeEnum.StudentsResultTable;
-                StudentsList = _excelDataReader.StudentListFromExcelTable();
-            }
             else
-            {
                 TableType = (int)TableTypeEnum.StudentsLogsTable;
-                LogsList = _excelDataReader.LogListFromExcelTable();
-            }
+        }
+
+        private void SetStudentsList(ObservableCollection<Student> newStudentsList)
+        {
+            TableType = (int)TableTypeEnum.StudentsResultTable;
```

Comment:
Is there any possible scenario where the cast to int fails here? If there is how exactly would it fail?